tutorials/tutorial-howto-visualize-missing-data-grafana.md

```diff
@@ -0,0 +1,183 @@
+# Tutorial: How to visualize and aggregate missing time-series data in Grafana
+
+>:TOPLIST:
+> - [Introduction](#introduction)
+> - [Prerequisites](#prereqs)
+> - [Step 0 - Load your time-series data into TimescaleDB and simulate 
+missing data (optional)](#step0)
+> - [Step 1 - Plot the dataset and confirm missing data](#step1)
+> - [Step 2 - Interpolate (fill in) the missing data](#step2)
+> - [Step 3 - Aggregate across a larger time window](#step3)
+> - [Next Steps](#next-steps)
+
+### Introduction [](introduction)
+[Grafana][grafana-external] is an open-source visualization tool.
```

Comment:
The change from what grafana is to what this tutorial is about feels quite abrupt and could use another sentence